packages/modules/src/modules/banners/choiceCardsBanner/ChoiceCardsBanner.stories.tsx

```diff
@@ -116,8 +116,11 @@ ChoiceCardsBannerBlue.args = {
     tracking,
     choiceCardAmounts: {
         testName: 'Storybook_test',
-        variantName: 'Control',
-        amounts: {
+        variantName: 'CONTROL',
+        defaultContributionType: 'MONTHLY',
+        displayContributionType: ['ONE_OFF', 'MONTHLY', 'ANNUAL'],
+        // displayContributionType: ['ONE_OFF', 'ANNUAL'],
```

Comment:
Should this commented line be removed?

Reply:
Yes - I used it as a convenient shortcut when demoing to Marketing a few weeks ago. similar with other commented out lines in storybook story files

Reply:
Done